### ISSUE TITLE
Fix a11y issues identified by axe testing of Front of House

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5245]
Axe testing has been trialed during the recent a11y audit. (see openlibhums/scripts#2).

This has identified violations across the front of house.  As automated tests are blunt, some of these may turn out not to require work, others may have been missed, however, as a starting point we should have a test-and-fix piece of work.  I have not included the specific results here, because the tests can easily be rerun and any changes to the codebase between audit and when this issue is worked on would then be taken into account.  

1. how to test with axe is documented with the test scripts. 
2. Run test, investigate errors, fix, run again etc. 

For discussion at Backlog refinement:
- which theme(s) to prioritise
- Axe tests against WCAG criterion, and for best practice, are we targeting or prioritising specific criteria?
